src/taipy/core/pipeline/_pipeline_manager.py

```diff
@@ -107,23 +105,17 @@ def _get_or_create(
         else:
             owner_id = None
 
-        filters = cls._build_filters_with_version(None)
-        if pipelines_from_owner := cls._repository._get_by_config_and_owner_id(  # type: ignore
-            str(pipeline_config.id), owner_id, filters
-        ):
-            return pipelines_from_owner
-
         version = cls._get_latest_version()
         pipeline = Pipeline(
-            str(pipeline_config.id),
-            dict(**pipeline_config._properties),
+            {},
```

Comment:
Please add the name as a property:
```suggestion
            {"name": pipeline_name},
```